public/app/features/apiserver/client.ts

```diff
@@ -90,3 +97,13 @@ export class ScopedResourceClient<T = object, K = string> implements ResourceCli
       .join(',');
   }
 }
+
+// add the origin annotations so we know what was set from the UI
+function setOriginAsUI(meta: Partial<ObjectMeta>) {
+  if (!meta.annotations) {
+    meta.annotations = {};
+  }
+  meta.annotations.AnnoKeyOriginName = 'UI';
```

Comment:
As discussed with @fabrizia a while ago.  This way we can better identify what came from the UI vs all the other ways things can get into the system